orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -636,6 +636,14 @@
     <service>
         <role>NIFI</role>
         <param name="useTwoWaySsl" value="true" />
+        <param>
+            <name>httpclient.connectionTimeout</name>
+            <value>2m</value>
```

Comment:
Don't we have a 60 sec timeout on nginx anyway?

Reply:
I think @biharitomi increased it recently, although I'm not sure on which branch.

Reply:
I think @akanto is right, I only increased the cluster proxy client and NGiNX timeouts.

Reply:
But we CB communicate with CM directly through NGiNX on port 9443 and the Knox timeout should not be increased for the deploy client config calls at least.